setup.py

```diff
@@ -110,48 +138,72 @@ def run_tests(self):
             raise SystemExit(1)
 
 
-def _get_version():
-    from os.path import abspath, dirname, join
-    filename = join(dirname(abspath(__file__)), 'bibliopixel', 'VERSION')
-    return open(filename).read().strip()
+def open_local(paths, mode='r', encoding='utf8'):
+    path = os.path.join(
+        os.path.abspath(os.path.dirname(__file__)),
+        *paths
+    )
+
+    return codecs.open(path, mode, encoding)
+
+
+with open_local(['bibliopixel', '__init__.py'], encoding='utf8') as fp:
+    try:
+        init = fp.read()
+        __version__ = re.search(r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]', init).group(1)
```

Comment:
There has to be a better way!

Also, what's with all these `__` variables??  You clearly have some master plan where you're adding a lot of `__` variables that have no meaning to Python.

This doesn't sit well with me.  Right now, if a variable or file starts with `__`, then this means something to Python.  That's a really useful convention and I don't want to drop it without a good reason.

Reply:
As mentioned previously, it was mainly used for epydoc related generation.
I've kept with it in various other projects, but can't find much information about the dunder metadata officially.